src/seffnet/chemical_similarities.py

```diff
@@ -234,60 +249,67 @@ def get_combined_graph_similarity(
         return nx.read_edgelist(DEFAULT_GRAPH_WEIGHTED_PATH)
     elif not rebuild and os.path.exists(DEFAULT_GRAPH_PATH):
         return nx.read_edgelist(DEFAULT_GRAPH_PATH)
-    if type(fullgraph_path) == pybel.struct.graph.BELGraph:
+
+    if isinstance(fullgraph_path, BELGraph):
         fullgraph_without_chemsim = fullgraph_path
     else:
         fullgraph_without_chemsim = pybel.from_pickle(fullgraph_path)
-    if type(chemsim_graph_path) == pybel.struct.graph.BELGraph:
+
+    if isinstance(chemsim_graph_path, BELGraph):
         chemsim_graph = chemsim_graph_path
     else:
         chemsim_graph = pybel.from_pickle(chemsim_graph_path)
 
+    fullgraph_with_chemsim = fullgraph_without_chemsim + chemsim_graph
+    pybel.to_pickle(fullgraph_with_chemsim, pickle_graph_path)
+
     mapping_df = pd.read_csv(
         mapping_file,
         sep="\t",
         dtype={'identifier': str, 'node_id': str},
         index_col=False,
     )
-    fullgraph_with_chemsim = fullgraph_without_chemsim + chemsim_graph
-    pybel.to_pickle(fullgraph_with_chemsim, pickle_graph_path)
+
     relabel_graph = {}
-    for ind, row in mapping_df.iterrows():
-        if row['namespace'] == PUBCHEM_NAMESPACE:
-            relabel_graph[pybel.dsl.Abundance(namespace=PUBCHEM_NAMESPACE, identifier=row['identifier'])] = \
-                row['node_id']
-        elif row['namespace'] == UNIPROT_NAMESPACE:
-            relabel_graph[pybel.dsl.Protein(namespace=UNIPROT_NAMESPACE, identifier=row['identifier'],
-                                            name=row['name'])] = row['node_id']
+    for node_id, namespace, identifier, name in mapping_df[['node_id', 'namespace', 'identifier', 'name']].values:
```

Comment:
using `df.values` eliminates the index, and slicing it allows us to explicitly take the variables out of the row we're using and assign them